DataFormats/Detectors/TPC/src/DataFormatsTPCLinkDef.h

```diff
@@ -22,7 +22,8 @@
 #pragma link C++ class o2::TPC::ClusterNativeContainer + ;
 #pragma link C++ class std::vector < o2::TPC::ClusterNative > +;
 #pragma link C++ class std::vector < o2::TPC::ClusterNativeContainer > +;
-#pragma link C++ class std::vector < o2::TPC::ClusterHardwareContainer8kb > +;
+#pragma link C++ class std::vector < o2::TPC::ClusterHardware > +;
+//#pragma link C++ class std::vector < o2::TPC::ClusterHardwareContainer8kb > +;
```

Comment:
Why do disable this dictionary? Might be the reason of your difficulties with RegisterAny.

Reply:
on the contrary, with this line, the dictionary is not found even for the plain pointer